migrations/versions/7-notifications_create_on.py

```diff
@@ -0,0 +1,38 @@
+"""Notifications 'create_on' migration
+
+Revision ID: e3d06397960d
+Revises: 2fcadbe2f0ad
+Create Date: 2024-03-17 19:19:14.297195
+
+"""
+
+from collections.abc import Sequence
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "e3d06397960d"
```

Comment:
The revision should be the one of phonebook
```suggestion
revision: str = "6afc765adaa2"
```